src/ShopCertification/PhpRequester.php

```diff
@@ -34,30 +34,29 @@ public function request($action, array $data)
                 throw new RequesterException('Failed to serialize data into JSON. Data: ' . var_export($data, true));
             }
 
-            $urlParts = parse_url($this->endpoint->getUrl() . $action);
-            $scheme = isset($urlParts['scheme']) && $urlParts['scheme'] === 'https' ? 'ssl://' : '';
-            $port = isset($urlParts['port']) ? $urlParts['port'] : ($urlParts['scheme'] === 'https' ? 443 : 80);
-            $fp = fsockopen($scheme . $urlParts['host'], $port, $errorNo, $errorMessage);
+            $options = [
+                'http' => [
+                    'method' => "POST",
+                    'header' => "Content-Type: application/json\r\n" . sprintf("Content-Length: %d\r\n", strlen($json)),
+                    'content' => $json,
+                    'ignore_errors' => true
+                ]
+            ];
+
+            $context = stream_context_create($options);
+            $fp = fopen($this->endpoint->getUrl() . $action, 'r', false, $context);
+
             if ($fp === false) {
-                throw new RequesterException(sprintf('fsockopen failed: [%d] %s', $errorNo, $errorMessage));
+                throw new RequesterException(sprintf('fopen failed'));
```

Comment:
Tyjo tady musí být nějaká možnost jak zjistit bližší info, něco jako to bylo předtím ...